src/components/views/emojipicker/ReactionPicker.js

```diff
@@ -24,7 +24,7 @@ class ReactionPicker extends React.Component {
         mxEvent: PropTypes.object.isRequired,
         onFinished: PropTypes.func.isRequired,
         closeMenu: PropTypes.func.isRequired,
-        reactions: PropTypes.object.isRequired,
+        reactions: PropTypes.object,
```

Comment:
hmm?

Reply:
this comes from MessageActionBar which has PropTypes.object, ReactionPicker has null-guards yet the PropType says its Required. MessageActionBar defaults to undefined so causes proptypes error

Reply:
ah, alright then